src/platform/plugins/shared/controls/public/controls/data_controls/options_list_control/selections_manager.ts

```diff
@@ -21,22 +22,24 @@ function areSelectedOptionsEqual(
 }
 
 export const selectionComparators: StateComparators<
-  Pick<OptionsListControlState, 'exclude' | 'existsSelected' | 'selectedOptions'>
+  Pick<OptionsListControlState, 'exclude' | 'existsSelected' | 'selectedOptions' | 'sort'>
```

Comment:
I moved `sort` into here because it is **technically** a selection, since it can be saved into the control 🤷